README.md

```diff
@@ -1 +1,115 @@
 # example-golang
+
+An example repository to demonstrate Pants's experimental Golang support.
+
+See [] for some unique benefits Pants brings to Golang repositories, and see
+[pantsbuild.org/docs/go-overview](https://www.pantsbuild.org/v2.8/docs/go-overview) for much more detailed
+documentation.
+
+This is only one possible way of laying out your project with Pants. See 
+[pantsbuild.org/docs/source-roots#examples](https://www.pantsbuild.org/docs/source-roots#examples) 
+for some other example layouts.
+
+Note: for now, Pants only supports repositories using a single `go.mod`. Please comment on 
+[#13114](https://github.com/pantsbuild/pants/issues/13114) if you need support for greater 
+than one `go.mod` so that we can prioritize adding support.
+
+# Running Pants
+
+You run Pants goals using the `./pants` wrapper script, which will bootstrap the
+configured version of Pants if necessary.
+
+# Goals
+
+Pants commands are called _goals_. You can get a list of goals with
+
+```
+./pants help goals
+```
+
+Most goals take arguments to run on. To run on a single directory, use the directory name with 
+`:` at the end. To recursively run on a directory and all its subdirectories, add `::` to the 
```

Comment:
Maybe mention that we are revisiting the dir spec syntax in 2.9?

Reply:
Based on discussion this week, it's no longer a firm commitment that we'll fix this in 2.9